src/reducers/tasks.js

```diff
@@ -0,0 +1,97 @@
+import { combineReducers } from 'redux';
+import union from 'lodash.union';
+
+const byId = (state = {}, action) => {
+  switch(action.type) {
+    case 'tasks/FETCH_ALL_TASKS_SUCCESS':
+    case 'tasks/CREATE_TASK_SUCCESS':
+    case 'tasks/UPDATE_TASK_SUCCESS':
+      return {
+        ...state,
+        ...action.response.entities.tasks,
+      };
+    default:
+      return state;
+  }
+};
+
+const allIds = (state = [], action) => {
+  switch(action.type) {
+    case 'tasks/FETCH_ALL_TASKS_SUCCESS':
+      return union(state, action.response.result.tasks);
+    case 'tasks/CREATE_TASK_SUCCESS':
+    case 'tasks/UPDATE_TASK_SUCCESS':
+      return union(state, [action.response.result]);
+    default:
+      return state;
+  }
+};
+
+const currentId = (state = null, action) => {
+  switch(action.type) {
+    case 'tasks/SET_TASK_ID':
+      return action.idtask;
+    default:
+      return state;
+  }
+};
+
+const isTasksAction = /^tasks/;
+
+const isRequest = /_REQUEST$/;
```

Comment:
instead of using a regex and naming pattern, maybe add a property to action objects that explicitly identifies them as requests, success, or failure?